package.json

```diff
@@ -80,17 +87,37 @@
     ]
   },
   "scripts": {
-    "start": "concurrently npm:server npm:dev",
     "dev": "react-scripts start",
     "build": "node ./scripts/build.js",
     "test": "jest",
     "eject": "react-scripts eject",
     "lint": "eslint src/**/*.{ts,tsx}",
     "server": "node server.js",
-    "test:ci": "jest --ci --runInBand --reporters=default --reporters=jest-junit --coverage",
```

Comment:
I wonder where this went to. I see other test:ci entries on the right hand side but they don't mention jest.